projects/canopy/src/lib/carousel/carousel.stories.ts

```diff
@@ -0,0 +1,82 @@
+</lg-carousel-item>
+
+<lg-carousel-item>
+  <h3>Carousel item 3</h3>
+  <p>Carousel items scale vertically to the largest item in the carousel and scale horizontally to the width of the parent container.</p>
+  <p>Carousels are built to work with varied content sizes but will work best if the content of each carousel item is of a similar size</p>
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit maiores, dolor suscipit ullam omnis corrupti laboriosam, amet, rem fugiat id deleniti ipsum possimus. At assumenda, asperiores dolorem temporibus quaerat libero quos blanditiis quod in? Magni laborum tempora, laudantium facilis ut sunt totam beatae repellat doloribus blanditiis sed repellendus maiores necessitatibus iste ex quod voluptate? Nesciunt, libero officiis labore molestias enim doloribus id assumenda dicta numquam fuga velit reiciendis quae praesentium soluta eaque exercitationem? Ullam iste culpa accusantium tenetur repellendus sit sunt error nisi dicta deleniti cum amet, voluptates soluta et laudantium quis.</p>
+</lg-carousel-item>
+
+<lg-carousel-item>
+  <h3>Carousel item 4</h3>
+  <p>Carousel items scale vertically to the largest item in the carousel and scale horizontally to the width of the parent container.</p>
+  <p>Carousels are built to work with varied content sizes but will work best if the content of each carousel item is of a similar size</p>
+  <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit maiores, dolor suscipit ullam omnis corrupti laboriosam, amet, rem fugiat id deleniti ipsum possimus. At assumenda, asperiores dolorem temporibus quaerat libero quos blanditiis quod in? Magni laborum tempora, laudantium facilis ut sunt totam beatae repellat doloribus blanditiis sed repellendus maiores necessitatibus iste ex quod voluptate? Nesciunt, libero officiis labore molestias enim doloribus id assumenda dicta numquam fuga velit reiciendis quae praesentium soluta eaque exercitationem? Ullam iste culpa accusantium tenetur repellendus sit sunt error nisi dicta deleniti cum amet, voluptates soluta et laudantium quis.</p>
+</lg-carousel-item>
+`;
+
+function props() {
+  const slideDurationOptions = {
+    range: true,
+    min: 0,
+    max: 5000,
+    step: 100,
+  };
+  return {
+    headingLevel: select('headingLevel', [1, 2, 3, 4, 5, 6], 3),
```

Comment:
Is it worth making the labels more readable i.e. 'Heading level' rather than 'headingLevel'

Reply:
Fixed